frontend/webapp/core/fetch_hijack.py

```diff
@@ -6,15 +6,15 @@
 log = logging.getLogger("webapp_logger")
 
 
-def get_hijack_by_key(hijack_key):
+def check_if_hijack_exists(hijack_key):
     try:
         log.debug("send request for total get_hijack_by_key")
         url_ = API_URI + "/view_hijacks?key=eq." + hijack_key
         response = requests.get(url=url_)
         raw_json = response.json()
         log.debug("response: {}".format(raw_json))
-        if raw_json:
-            return raw_json[0]
+        if len(raw_json):
```

Comment:
"if raw_json" is sufficient, please change

Reply:
done